lib/utils.js

```diff
@@ -59,6 +60,7 @@ utils.instantiate = function(ast, options) {
   } else {
     snapdragon = new Snapdragon(options);
   }
+  snapdragon.use(capture())
```

Comment:
Should I only add capture when doing new Snapdragon() here maybe? But I guess using `snapdragon.use(capture())` multiple times on the same snap instance does not harm anything.

Reply:
I'll leave that up to you. I can go either way, and you have thought it though.